.github/workflows/update-and-process-tranco.yml

```diff
@@ -0,0 +1,288 @@
+            echo "One or more output files are not defined in manifest.json. Please update manifest.json first."
+            rm $TEMP_FILE
+            exit 1
+          fi
+          
+          rm $TEMP_FILE
+      
+      - name: Process Tranco CSV
+        id: process
+        run: |
+          echo "Processing Tranco CSV with configuration: $CONFIG"
+          
+          # Check if tranco.csv exists
+          if [ ! -f "tranco.csv" ]; then
+            echo "Error: tranco.csv file not found"
+            exit 1
+          fi
+          
+          # Parse the JSON configuration and process each output
+          echo $CONFIG | jq -c '.[]' | while read -r config; do
+            count=$(echo $config | jq -r '.count')
+            filename=$(echo $config | jq -r '.filename')
+            
+            if [ -z "$count" ] || [ -z "$filename" ]; then
+              echo "Skipping invalid configuration: $config"
+              continue
+            fi
+            
+            # Get exactly the requested number of lines from the file
+            head -n $count tranco.csv > "$filename"
+            
+            lines=$(wc -l < "$filename")
+            echo "Successfully created $filename with $lines rows"
+          done
+      
+      - name: Configure Git
+        run: |
+          git config --local user.email "hello@sublimesecurity.com"
+          git config --local user.name "Tranco Process Bot"
+      
+      - name: Create and push branch
+        id: create-branch
+        run: |
+          # Create a unique branch name with timestamp
+          BRANCH_NAME="tranco_update-${{ steps.date.outputs.today }}-${{ steps.date.outputs.timestamp }}"
+          echo "branch_name=$BRANCH_NAME" >> $GITHUB_OUTPUT
+          
+          git checkout -b "$BRANCH_NAME"
+          git add tranco.csv
+
+          # Parse the configuration to get the filenames
+          echo $CONFIG | jq -c '.[]' | while read -r config; do
+            filename=$(echo $config | jq -r '.filename')
+            # Add each generated file individually
+            git add "$filename"
+          done
+
+          git commit -m "Update Tranco list for ${{ steps.date.outputs.today }} (ID: ${{ steps.tranco-id.outputs.id }})"
+          git push origin "$BRANCH_NAME"
+      
+      - name: Create Pull Request
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh pr create \
+            --title "Update Tranco list and derived files - ${{ steps.date.outputs.today }}" \
+            --body "This PR updates the Tranco top 1 million domains list and all derived files.
+            
+            - Date: ${{ steps.date.outputs.today }}
+            - Tranco List ID: ${{ steps.tranco-id.outputs.id }}
+            - List URL: https://tranco-list.eu/list/${{ steps.tranco-id.outputs.id }}
+            - Automated update via GitHub Actions" \
+            --head "$BRANCH_NAME" \
+            --base "master"
```

Comment:
I'd recommend adding `--reviewer @sublime-security/detection` to help give the PRs visibility.